Formula/vulkan-loader.rb

```diff
@@ -31,6 +32,8 @@ def install
     system "cmake", "-S", ".", "-B", "build",
                     "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].opt_prefix}",
                     "-DFALLBACK_DATA_DIRS=#{HOMEBREW_PREFIX}/share:/usr/local/share:/usr/share",
+                    "-DCMAKE_INSTALL_SYSCONFDIR=#{HOMEBREW_PREFIX}/etc",
+                    "-DFALLBACK_CONFIG_DIRS=#{HOMEBREW_PREFIX}/etc/xdg:/etc/xdg",
```

Comment:
```suggestion
                    "-DCMAKE_INSTALL_SYSCONFDIR=#{etc}",
                    "-DFALLBACK_CONFIG_DIRS=#{etc}/xdg:/etc/xdg",
```
`etc == HOMEBREW_PREFIX/"etc"`